install.go

```diff
@@ -10,11 +10,11 @@ import (
 func InstallRHMAP(conf Config) {
 	log.Println("Running rhmap-ansible installer...")
 
-	var cmd = sh.Command("sh", "-c", fmt.Sprintf("docker run -v %s/inventories:/opt/app-root/src/inventories"+
+	var cmd = sh.Command("sh", "-c", fmt.Sprintf("docker run -v %s/inventory-templates:/opt/app-root/src/inventory-templates"+
```

Comment:
has the expected dir in the docker container changed too?

Reply:
@david-martin yep it has - used to be in `inventories` in rhmap-ansible, but these have moved to a new repo now.